### PR TITLE
convergence tracker for netpol

### PR DESCRIPTION
**Pull request by venkataanil**
convergencetracker image is used in network policy scale testing. These files are taken from Nadia's convergence tracker[1]. It moniots OVS flows and measures when OVS flows gets stabilized and report this as a metric.

[1] https://github.com/npinaeva/k8s-netpol-scale/tree/main/kube-burner-workload/openshift/openflow-tracker

### CODE REVIEW

**Comment by rsevilla87** [comment]
@venkataanil , there're some conflicts

File: convergencetracker/Containerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM python:slim
```
Comment (krishvoor) [inline convergencetracker/Containerfile:1]: Hi @venkataanil I would like for us to work with UBI8/9 Images, can you change this to below:

```
FROM registry.access.redhat.com/ubi9/ubi-minimal

RUN microdnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs \
    python3.11 python3.11-devel python3.11-pip \
    && microdnf clean all --enablerepo='*'

# PYTHONDONTWRITEBYTECODE 1 : disable the generation of .pyc
# PYTHONUNBUFFERED 1 : force the stdout and stderr streams to be unbuffered
# PYTHONCOERCECLOCALE 0, PYTHONUTF8 1 : skip legacy locales and use UTF-8 mode
ENV PYTHONDONTWRITEBYTECODE=1 \
    PYTHONUNBUFFERED=1 \
    PYTHONCOERCECLOCALE=0 \
    PYTHONUTF8=1 \
    PYTHONIOENCODING=UTF-8 \
    LANG=en_US.UTF-8 \
    PIP_NO_CACHE_DIR=off
```
Reference: https://github.com/cloud-bulldozer/images/blob/main/netpol-scraper/Dockerfile#L1-L16

Comment (venkataanil) [reply convergencetracker/Containerfile:1]: I will give a try

File: README.md [hunk 2]
```diff
@@ -14,3 +14,4 @@ Container Images for OCP Perf&Scale
 - nginxecho: quay.io/cloud-bulldozer/nginxecho:latest
 - netpolvalidator: quay.io/cloud-bulldozer/netpolvalidator:latest
 - netpolproxy: quay.io/cloud-bulldozer/netpolproxy:latest
+- convergencetracker: quay.io/cloud-bulldozer/convergencetracker:latest
```
Comment (krishvoor) [inline README.md:17]: @rsevilla87: Can you help create `quay.io/cloud-bulldozer/convergencetracker:latest` with similar ROBOT account privileges as `https://quay.io/repository/cloud-bulldozer/netpolproxy` Repo

File: convergencetracker/Containerfile (hunk 1 above)
Comment (krishvoor) [reply convergencetracker/Containerfile:1]: Based on more digging with @venkataanil  `openvswitch-switch, ovn-central` rpms are not found in UBI8/9 by default and might need us to build those rpms in the image (which might bump the size of container and add to overall bulid/publish process). I'm ok with `python:slim` which in itself is multi-arch in nature.

**Review by krishvoor — APPROVED** [review]
/lgtm

File: README.md (hunk 2 above)
Comment (krishvoor) [reply README.md:17]: Thanks for @rsevilla87

**Pull request merged**